### PR TITLE
Title has been changed

### PR DESCRIPTION
**Pull request by dynamiteweb**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<title>Coming Soon</title>
+<title>Leicester Kofukan Karate\</title>
```
Comment (kane-menicou) [inline index.html]: This has a slash on the end of it

Comment (dynamiteweb) [inline index.html]: Okay, thanks I am going to fix this in the next commit! :)

Comment (dynamiteweb) [inline index.html]: This has been changed :)

**Comment by kane-menicou** [comment]
This is okay to merge

**Pull request merged**